resources/views/events/github/team/deleted.blade.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+/**
+ * @var $payload object
+ */
+
+?>
+
+⚡ <b>{!! __('tg-notifier::events/github/team.deleted.title') !!}</b> 🎊
```

Comment:
Can you please change the trash icon for this line?

```suggestion
🗑 <b>{!! __('tg-notifier::events/github/team.deleted.title') !!}</b> 🎊
```